tools/topology/topology2/cavs-sdw-src-gain-mixin.conf

```diff
@@ -93,14 +93,13 @@ Object.Pipeline {
 		}
 	]
 
-	mixout-gain-dai-copier-playback [
+	mixout-gain-alh-dai-copier-playback [
 			{
 			index 2
 
-			Object.Widget.copier.1 {
+			Object.Widget.alh-copier.1 {
 				stream_name	$SDW_PLAYBACK_PCM
-				dai_type	"ALH"
-				copier_type	"ALH"
+				dai_index	0
```

Comment:
do we need the dai_index?

Reply:
Yes, we do need the dai_index. When there's aggregation, the dai_index would be set based on the index of the DAI in the aggregated group. When there' sno aggregation, it will just be 0